src/Controls/tests/DeviceTests/Resources/Raw/HybridTestRoot/scripts/HybridWebView.js

```diff
@@ -91,20 +91,49 @@
         }
     },
 
-    "__InvokeJavaScript": function __InvokeJavaScript(taskId, methodName, args) {
```

Comment:
These changes are backwards/forwards compatible. Before this PR, an error just never calls back into C#. All the changes are in the new function that catches exceptions and calls back into .NET. If you have the old JS, nothing will change.